main.go

```diff
@@ -1,16 +1,36 @@
 package main
 
 import (
-	"io"
-	"log"
+	"fmt"
 	"net/http"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	http_router "github.com/julienschmidt/httprouter"
 )
 
 func main() {
-	helloHandler := func(w http.ResponseWriter, req *http.Request) {
-		io.WriteString(w, "Server is running...")
+	port := "8080"
```

Comment:
Can try setting such infrastructure-related configs as an environment variable.
Can use the [os](https://golang.org/pkg/os/) to get environment variables  using the function `os.Getenv("VARIABLE_NAME")`

Reply:
I have raised separate pull request for this suggestion:
https://github.com/ashwinspg/explore-golang/pull/2